src/ray/raylet/local_object_manager.cc

```diff
@@ -613,6 +622,18 @@ void LocalObjectManager::RecordMetrics() const {
   ray::stats::STATS_spill_manager_request_total.Record(spilled_objects_total_, "Spilled");
   ray::stats::STATS_spill_manager_request_total.Record(restored_objects_total_,
                                                        "Restored");
+
+  ray::stats::STATS_object_store_memory.Record(
+      spilled_bytes_current_,
+      {{"Location", ray::rpc::ObjectLocation_Name(ray::rpc::ObjectLocation::SPILLED)}});
+}
+
+size_t LocalObjectManager::GetCurrentSpilledCount() const {
```

Comment:
Is it test only? Can you comment it? (if it is test only, we can also just use friend class)

Reply:
oh yeah - test only for now.